Added Credits section in README.md

miriambrus opened the pull request:
- Added Credits section in README.md https://github.com/Azure/opendigitaltwins-smartcities/tree/workingbranch-smartcitiesontology#credits
- Attributed credits for the NGSI-LD data models in this repository have been adapted to DTDL from the open source [Smart Models repository for Smart Cities](https://github.com/smart-data-models) developed by FIWARE Foundation and TM Forum. For more information on the Smart Data Models please visit [https://smartdatamodels.org](https://smartdatamodels.org/).
- Removed crossdomain folder introduced by https://github.com/Azure/opendigitaltwins-smartcities/pull/41/commits/88c929e9e148c1a767ec3ff23c1ae7affcb96f8d
- Updated device.json in Ontology/crossdomain with fix to dateObserved from 92507fe. 
- Updated naming of Device and Device Model entities to PascalCase. Upadted naming of deviceCategory component to camelCase

ACSirus commented:
Hi @miriambrus. To be 100% correct, the link behind "Smart Models repository for Smart Cities" should be changed to
"https://github.com/smart-data-models" as I got some of my original models from another repository that is not directly referenced through the current link. E.g NoiseLevelObserved, AirQualityObserved,...

miriambrus commented:
> Hi @miriambrus. To be 100% correct, the link behind "Smart Models repository for Smart Cities" should be changed to
> "https://github.com/smart-data-models" as I got some of my original models from another repository that is not directly referenced through the current link. E.g NoiseLevelObserved, AirQualityObserved,...

Makes sense @ACSirus, updating it

miriambrus commented:
- Fixed with @ACSirus comment to point to https://github.com/smart-data-models on Credits
- Removed crossdomain folder introduced by https://github.com/Azure/opendigitaltwins-smartcities/pull/41/commits/88c929e9e148c1a767ec3ff23c1ae7affcb96f8d
- Updated device.json in Ontology/crossdomain with fix to dateObserved from 92507fe. 
- Updated naming of Device and Device Model entities to PascalCase. Upadted naming of deviceCategory component to camelCase
